field.go

```diff
@@ -18,6 +18,10 @@ type DateFields struct {
 	UpdatedAt time.Time `json:"updated_at" bson:"updated_at"`
 }
 
+type VersionField struct {
+	Version int `json:"_v" bson:"_v"`
```

Comment:
What do you think about changing the struct field's name from `Version` to `V` too?

Reply:
I dont think V is a very good name for a field. I would keep 'Version' personally given having VersionField is an opt in now and if you dont like it, you can always create your own VersionField. 
But if you really dont like Version, we could also use '_Version' which reflects nicely the serialized version '_v'.

Reply:
The reason that I think it's good to change this field's name is the same as the reason that we changed tag values, I think users don't need to the Version field directly and just include it in their structs, so maybe it's good to let them have the `Version` name for their own fields.
`_Version` is good but it's private, I think a public field could be better.

Reply:
Ok, I have changed it to Version_ which is public and also has the underscore to indicate it is a special field. Let me know what you think.
Also, dont forget we have removed VersionField from the default model so it is relatively easy to have your own VersionField defined with the name you want.